docs/source/user_guide/training/metric.rst

```diff
@@ -65,46 +59,48 @@ Example:
     tensor(1.9999)
     >>> l = dinv.loss.MCLoss(metric=dinv.metric.SSIM(train_loss=True, reduction="mean")) # Use SSIM for training
 
+.. _distortion-metrics:
+
 Distortion metrics
 ------------------
+Distortion metrics are used to measure the difference between the original ``x`` and the reconstructed image ``x_net``.
 
-We implement popular distortion metrics
-(see `The Perception-Distortion Tradeoff <https://openaccess.thecvf.com/content_cvpr_2018/papers/Blau_The_Perception-Distortion_Tradeoff_CVPR_2018_paper.pdf>`_
-for an explanation of distortion vs perceptual metrics):
+.. list-table:: Distortion Metrics
+   :header-rows: 1
 
-.. autosummary::
-   :toctree: stubs
-   :template: myclass_template.rst
-   :nosignatures:
+   * - **Metric**
+     - **Definition**
 
-        deepinv.loss.metric.MSE
-        deepinv.loss.metric.NMSE
-        deepinv.loss.metric.MAE
-        deepinv.loss.metric.PSNR
-        deepinv.loss.metric.SSIM
-        deepinv.loss.metric.QNR
-        deepinv.loss.metric.L1L2
+   * - :class:`deepinv.loss.metric.MSE`
+     - :math:`\text{MSE}(x, \hat{x}) = \frac{1}{n} \sum_{i=1}^n (x_i - \hat{x}_i)^2`
```

Comment:
```suggestion
     - :math:`\text{MSE}(\hat{x},x) = \frac{1}{n} \sum_{i=1}^n (x_i - \hat{x}_i)^2`
```
You seem to have missed this correction out

Reply:
sorry for the miss, it should be fixed now